README.md

```diff
@@ -2,10 +2,10 @@
 
 # LeaderBoard
 
-> LeaderBoard functionality
```

Comment:
- [ ] README file should contain all that's needed for running the app with the API(s) it consumes. It should have instructions about where and how to generate any necessary keys, credentials, how to set, store, and generate them (if necessary)

Reply:
Yes but that was not part of requirements I could just pul a link to their documentations, would it do ?